.env

```diff
@@ -0,0 +1,2 @@
+NODE_PATH='src/'
+BLOCKCYPHER_TOKEN=539e6971dd0342ba988e3e4bc609c809
```

Comment:
this should also be removed: the token is passed as a CLI paramter to this service